cmd/integration/integration.go

```diff
@@ -435,11 +435,14 @@ containers:
 }
 
 func runReplicationControllerTest(c *client.Client) {
+	t := time.Now()
 	clientAPIVersion := c.APIVersion().String()
 	data, err := ioutil.ReadFile("cmd/integration/" + clientAPIVersion + "-controller.json")
 	if err != nil {
 		glog.Fatalf("Unexpected error: %v", err)
 	}
+	glog.Infof("Done reading config file, took %v", time.Since(t))
```

Comment:
Don't glog lines include the time? Not that computing the elapsed time hurt anything.